cns/requestcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -231,7 +231,9 @@ func (crdRC *crdRequestController) initCNS() error {
 		}
 	}
 
-	return crdRC.CNSClient.InitCNSState(&ncRequest, podInfoByIP)
+	// Call cnsclient init cns passing those two things
+	return crdRC.CNSClient.ReconcileNCState(&ncRequest, podInfoByIP)
```

Comment:
Can you also rename the above two calls from InitCNSState to ReconcileNCState? lines 194 and 204